doc/lsst.ap.pipe/index.rst

```diff
@@ -28,18 +25,6 @@ Overview
 
    pipeline-overview
 
-.. _lsst.ap.pipe-using-gen2:
-
-Using lsst.ap.pipe in Gen 2
-===========================
-
-.. toctree::
-   :maxdepth: 1
-
-   getting-started-gen2
-   pipeline-tutorial-gen2
-   apdb
-
 .. _lsst.ap.pipe-using:
 
 .. _lsst.ap.pipe-using-gen3:
```

Comment:
I'd suggest also removing the "Command Line Task Reference" section starting on new line 64. If this package is now Gen 3 only, then it will never be needed.